### DISCUSSION TITLE
[A/L] (2021/22) Combinatoria - Esercizio 12

### DISCUSSION THREAD

**Discussion by matypist** [sapienzastudentsnetwork/metodi-matematici-per-l-informatica#188] [Esercizi A-L]
![image](https://user-images.githubusercontent.com/69389461/152253083-6b03e6fe-df5b-4766-82d5-d4b80969881d.png)

**Comment by Rurik-D** [comment]
Il problema così com'è posto non è del tutto chiaro.  Diamo per buono che n sia uguale a 20 e che a1 debba essere strettamente maggiore di 0 (altrimenti avremmo un unico intervallo "vuoto" a fronte degli altri dove la relazione è ben chiarita nella seconda parte del problema). Fatta questa doverosa premessa ecco il mio ragionamento:

Abbiamo 19 valori da divedere in 9 intervalli (0 e 20 sono fissati, l'esercizio si basa sulla definizione degli **a** compresi nell'intervallo [0,20] che faranno da spartiacque per definire gli intervalli).
Dal momento che ogni **a** deve essere strettamente minore del precedente, sappiamo che ogni intervallo avrà almeno un'unità di differenza tra le due **a** che lo compongono, quindi il numero di valori da dividere per i 9 intervalli scende a 10 (ogni intervallo parte già con un un'unità di differenza tra le due **a** all'interno).
La soluzione è semplicemente quella delle combinazioni con ripetizioni.

1. #soluzione = (10 + 9 -1 scegli 10) = (18 scegli 10) = (18 scegli 8)
2. #soluzione = (7 + 12 -1 scegli 7) = (18 scegli 7) = (18 scegli 11)
3. Considero:
        V = #numero_valori
         I = #numero_intervalli
    Formula_generale = ( (V - 2 - I) + I -1  scegli  V - 2 - I ) = ( V - 3  scegli I -1 )
